local/fs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Missing a newline here.

I suggest to add comments to make the intention very clear. How about:

```
# Ignore everything
*
# But not this file itself.
!.gitignore
```